charts/sampling/templates/_otelcol_processor_transform.river.txt

```diff
@@ -0,0 +1,69 @@
+{{- define "deployment.processor.transform.add_resource_attributes" -}}
+otelcol.processor.transform "add_resource_attributes_as_metric_attributes" {
+  // https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.processor.transform/
+  error_mode = "ignore"
+
+  metric_statements {
+    context    = "datapoint"
+    statements = [
+      "set(attributes[\"deployment.environment\"], resource.attributes[\"deployment.environment\"])",
+      "set(attributes[\"service.version\"], resource.attributes[\"service.version\"])",
+    ]
+  }
+
+  output {
+    metrics = [otelcol.processor.batch.default.input]
+  }
+}
+
+{{ end }}
+
+{{- define "deployment.processor.transform.add_k8s_attributes" -}}
+otelcol.processor.transform "add_k8s_attributes" {
+  // https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.processor.transform/
+  // Grafana Cloud Kubernetes monitoring expects Loki labels `cluster`, `pod`, and `namespace`
+  error_mode = "ignore"
+
+  log_statements {
+    context    = "resource"
+    statements = [
+      "set(attributes[\"pod\"], attributes[\"k8s.pod.name\"])",
+      "set(attributes[\"namespace\"], attributes[\"k8s.namespace.name\"])",
+      "set(attributes[\"loki.resource.labels\"], \"pod, namespace\")",
+    ]
+  }
+
+  trace_statements {
+    context    = "resource"
+    statements = [
+      "set(attributes[\"k8s.cluster.name\"], \"{{ .Values.k8s.cluster.name }}\")",
```

Comment:
in k8s-monitoring-helm they [always check](https://github.com/grafana/k8s-monitoring-helm/blob/8e8c0fb312508ced0f4c6b9b12fe02e1606f63cf/charts/k8s-monitoring/templates/agent_config/_traces.river.txt#L21
) if `{{ .Values.k8s.cluster.name }}` exists:
```
// Accessing a map with a key that does not exist will return nil.
  "set(attributes[\"k8s.cluster.name\"], \"{{ required ".Values.cluster.name is a required value. Please set it and try again." .Values.cluster.name }}\") where attributes[\"k8s.cluster.name\"] == nil",
```

we probably should do the same here

Reply:
Given that we're excluding any enrichment for now, I'll just remove this.